ch05.adoc

```diff
@@ -183,12 +183,16 @@ _This section has been superseded by the treatment of time series as a type of d
 [[grid-mappings-and-projections, Section 5.6, "Horizontal Coordinate Reference Systems, Grid Mappings, and Projections"]]
 === Horizontal Coordinate Reference Systems, Grid Mappings, and Projections
 
-When the coordinate variables for a horizontal grid are not longitude and latitude, the true latitude and longitude coordinates may be supplied via the **`coordinates`** attribute.  
-A __grid mapping variable__ is required if, in addition, it is desired to describe the mapping between the
-given coordinate variables and the true latitude and longitude coordinates, or to describe the
+A __grid mapping variable__ may be referenced by a data variable, to describe the mapping between the
```

Comment:
A __grid mapping variable__ may be referenced by a data variable to describe the mapping between the

Reply:
I prefer to keep some punctuation, as ths is a long sentence with a number of clauses

Reply:
@marqh That comma is grammatically wrong.